README.md

```diff
@@ -789,6 +798,30 @@ datetime                           fromAddress  fromAddressInExchange
 2019-04-30 15:17:28+00:00  0x876eabf441b2ee...                   True  0x1f4a90043cf2d...                False  0xc85892b9ef8c64...   20544.42975
 ```
 
+### Top Transfers
+
+Top transfers for the token of a given project, an address argument can be added as well, in the form of a key-value pair - ```address_selector="XYZ"```:
```

Comment:
The `addressSelector` in the API is a map that contains 2 keys: `address` (the one you refer as `addressSelector` here) and `transactionType` which can be `IN`, `OUT` or `ALL` (incoming/outgoing or all transactions, `ALL` by default).